composer.lock

```diff
@@ -4,25 +4,25 @@
         "Read more about it at https://getcomposer.org/doc/01-basic-usage.md#composer-lock-the-lock-file",
```

Comment:
The `composer.lock` should be in `.gitignore` for a library, since it leads to [impossible compatibility scenarios in CI](https://github.com/doctrine/instantiator/issues/33#issuecomment-319390586)